config/initializers/devise.rb

```diff
@@ -190,12 +190,12 @@
   # ==> Configuration for :recoverable
   #
   # Defines which key will be used when recovering the password for an account
-  # config.reset_password_keys = [:email]
+  config.reset_password_keys = [:login]
 
   # Time interval you can reset your password with a reset password key.
   # Don't put a too small interval or your users won't have the time to
   # change their passwords.
-  config.reset_password_within = 48.hours
+  config.reset_password_within = 7.days
```

Comment:
We should mention in the comment that the duration is currently hard-coded in the password reset email templates.

Let's also move this to ArchiveConfig.